src/mlpack/methods/ann/rnn_impl.hpp

```diff
@@ -363,14 +373,14 @@ EvaluateWithGradient(const arma::mat& /* parameters */,
     {
       outputLayer.Backward(std::move(boost::apply_visitor(
           outputParameterVisitor, network.back())),
-          std::move(arma::mat(responses.slice(rho - seqNum - 1).colptr(begin),
+          std::move(arma::mat(responses.slice(x - seqNum - 1).colptr(begin),
           responses.n_rows, batchSize, false, true)), std::move(error));
     }
 
     Backward();
     Gradient(std::move(
-        arma::mat(predictors.slice(rho - seqNum - 1).colptr(begin),
-        predictors.n_rows, batchSize, false, true)));
+    arma::mat(predictors.slice(x - seqNum - 1).colptr(begin),
+    predictors.n_rows, batchSize, false, true)));
```

Comment:
The starting of line should have 4 spaces when breaking a line, according to style guide